README.md

```diff
@@ -0,0 +1,40 @@
+### Documentation is included in the Documentation folder ###
+
```

Comment:
In End Process, add a check if a SystemException was caught and it  reached here with it, you would need to rethrow that caught SystemException otherwise the process will try to continue with the other workflows after an error in Init

Reply:
Added an if to check if SystemException is nothing and if it returns true then it attempts to do all the end process steps, else it throws the system exception again.